atom/browser/native_window_mac.mm

```diff
@@ -1078,7 +1081,15 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
       windowLevel = NSDockWindowLevel;
     }
   }
-  [window_ setLevel:windowLevel];
+
+  NSInteger newLevel = windowLevel + relativeLevel;
+  
+  if (newLevel >= 0 && newLevel <= maxWindowLevel) {
```

Comment:
Looks like there is a `kCGMinimumWindowLevelKey`, maybe that would be better to use than `0` here?